Documentation/Contributing/development-flow.md

```diff
@@ -17,20 +17,33 @@ don't hesitate to reach out to us on our [Slack](https://Rook-io.slack.com) dev
 
 From your browser navigate to [http://github.com/rook/rook](http://github.com/rook/rook) and click the "Fork" button.
 
-### Clone Your Fork
+### Clone Your Fork and add Upstream Remote
 
 Open a console window and do the following;
 
 ```console
 # Create the rook repo path
-mkdir -p $GOPATH/src/github.com/rook
+$ mkdir -p $GOPATH/src/github.com/rook
 
 # Navigate to the local repo path and clone your fork
-cd $GOPATH/src/github.com/rook
+$ cd $GOPATH/src/github.com/rook
 
 # Clone your fork, where <user> is your GitHub account name
 $ git clone https://github.com/<user>/rook.git
-cd rook
+
+# Add 'upstream' to the list of remotes
+$ cd rook
+$ git remote add upstream https://github.com/rook/rook.git
+
+# Fetch tags (needed for build process)
+$ git fetch --tags upstream
+```
+
+Now you should have at least `origin` and `upstream` remotes. You can also add other remotes to collaborate with other contributors.
+
+```console
```

Comment:
This should be above the fetch tag statement
As we would add first and then verify it
And then fetch